packages/metadata/src/index.ts

```diff
@@ -155,6 +157,26 @@ const extension: JupyterFrontEndPlugin<void> = {
         openMetadataWidget(args);
       }
     });
+
+    const schemas = await FrontendServices.getAllSchema();
+    console.log(schemas);
+    for (const schema of schemas) {
+      let icon = 'ui-components:text-editor';
+      if (schema.uihints && schema.uihints.icon) {
+        icon = schema.uihints.icon;
+      }
+      palette.addItem({
+        command: commandIDs.openMetadata,
+        args: {
+          label: `Show ${schema.display_name}`,
+          display_name: schema.display_name,
```

Comment:
We should switch to using `title` from the schema file.
```suggestion
          label: `Show ${schema.title}`,
          display_name: schema.title,
```